rules/python/tests/logging-debug-leak-test.py

```
@@ -0,0 +1,23 @@
+# Fixture: logging-debug-leak
+import logging
+
+user_password = "secret123"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "user_password"

(S105)